### PR TITLE
feat: implement requeue of messages.

### PR DESCRIPTION
**Pull request by xdanielsb**
solve wiremind/remoulade# 159
update docs

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -98,7 +98,13 @@ const parseGroup = rawGroup => {
   };
 };
 
+const requeue = id => {
```
Comment (antoinerabany) [inline src/api.js:101]: `messageID` would be more clear i think

File: src/components/CJobRow.vue [hunk 2]
```diff
@@ -16,10 +16,10 @@
       {{ isoWeekday | formatWeekDay }}
     </td>
     <td class="border px-4 py-2">
-      {{ args }}
+      <pre><div class="font-bold inline-block"></div>{{args | json}}</pre>
```
Comment (antoinerabany) [inline src/components/CJobRow.vue:19]: what is this div ? same below.

File: src/components/CMessageRow.vue [hunk 3]
```diff
@@ -70,8 +78,7 @@ export default {
       canCancel: true,
       isOpened: false,
       onError: false,
-      error: '',
-      txtBtnCancel: 'Cancel'
+      error: ''
```
Comment (antoinerabany) [inline src/components/CMessageRow.vue:81]: what is this error ? I think you should find a way to display a message if there is an error on requeue (example: if it is part of a pipeline).

Comment (xdanielsb) [reply src/components/CMessageRow.vue:81]: @antoinerabany  okay, let me fix it

File: src/components/CMessageRow.vue [hunk 4]
```diff
@@ -119,9 +128,30 @@ export default {
       };
       return colors[this.stateName] || 'black';
     },
+    showResponse(response) {
+      this.response = response;
+      this.canCancel = false;
```
Comment (antoinerabany) [inline src/components/CMessageRow.vue:133]: why canCancel=False ?

**Pull request merged**